inject-test/src/test/java/org/example/injectextension/WithExtnNamedSpyTest.java

```diff
@@ -20,7 +20,7 @@ class WithExtnNamedSpyTest {
   @Spy @Blue
   SomeStore blueStore;
 
-  @Spy @Named("green")
```

Comment:
Why change these string qualifiers to have "@" prefix like "green" to "@green"?

Reply:
It never made sense to me that the `@` symbol was omitted. It's an annotation qualifier so excluding `@` feels arbitrary

Reply:
it makes less sense when we allow annotation qualifiers since then it would look like `green(value=21)` instead of `@green(value=21)`

Reply:
I can update the PR to omit `@` again if you so desire, but I think it looks better this way

Reply:
The use of `@Named()` takes a string and there is no requirement for that string to start with `@` - and so it becomes unclear as to whether there is a breaking change to this. 

The changes in the PR typically indicate what tests were broken and needed to change in order to support the change (and instead I think that isn't clear). If we want to do a _"style change to the tests"_ then that should be in a separate PR.

Reply:
Now you only need `@` if the target bean uses a qualifier annotation, all other cases are the same